lib/commons/aria/get-owner-virtual.js

```diff
@@ -0,0 +1,54 @@
+/* global aria, dom */
+
+/**
+ * Get owner element of a provided virtualNode
+ * @param {Object} virtualNode virtual node
+ * @returns {HTMLElement | undefined}
+ */
+aria.getOwnerVirtual = function getOwnerVirtual(virtualNode) {
+	return getAriaOwner(virtualNode) || getOwner(virtualNode);
```

Comment:
Also thinking we should add options to this, allowing for the following:
- noAriaOwns
- ignoreNullRoleOwner

These could be passed in from the check options. Either add these now or open an issue for it.